.eslintrc.js

```diff
@@ -13,15 +23,18 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120 }],
     'vue/attribute-hyphenation': 'off',
+    'vue/multi-word-component-names': 'off',
     'vue/no-deprecated-filter': 'warn',
     'vue/no-deprecated-slot-scope-attribute': 'warn',
     'vue/no-deprecated-v-on-native-modifier': 'warn',
     'vue/no-v-for-template-key-on-child': 'warn',
     'vue/no-v-html': 'off',
+    'vue/no-v-model-argument': 'off',
```

Comment:
This is to fix lines 78 and 79 in `AddStaffNotationDialog.vue`. 
![addstaffnotaiondialog v-model directive require no argument](https://github.com/bcgov/business-filings-ui/assets/122301442/02d218be-9184-4bfd-a95a-29e10f3f13a0)

Reference: https://stackoverflow.com/questions/72952665/vue-no-v-model-argument-v-model-directives-require-no-argument-eslint-plugin-v

Reply:
Please change these to warnings, as we will have to fix this before going to Vue/Vuetify 3.